sqlgen/mysql_test.go

```diff
@@ -120,6 +119,16 @@ func TestSelectQuery(t *testing.T) {
 			OrderBy: "bar",
 		},
 	}, "SELECT foo, bar FROM foo ORDER BY bar", nil, t)
+
+	testQuery(&SelectQuery{
```

Comment:
as a sanity check, can we add a test that verifies the `FOR UPDATE` behavior works as expected? I see that this current test is strictly a syntax check

Reply:
Sure, I'll look into it 👍